lib/default.nix

```diff
@@ -5,13 +5,16 @@
  */
 let
 
-  callLibs = file: import file { inherit lib; };
+  fixedPoints = import ./fixed-points.nix {};
+
+  lib = fixedPoints.makeExtensible (self: let
+    callLibs = file: import file { lib = self; };
+  in rec {
```

Comment:
The indentation is broken from here.

Reply:
How would you format it?

Reply:
Don't use `rec`. Use `with self;` instead.